common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -534,12 +533,22 @@ def test_accordion_events(self):
                     'event': {
                         'target_name': 'Test Subsection 2,1 ',
                         'widget_placement': 'accordion',
+
                     }
                 },
             ],
-            outline_ui_events
+            selected_events
         )
 
+    def test_link_clicked_events(self):
+        self.course_nav.go_to_section('Test Section 1', 'Test Subsection 1,2')
```

Comment:
Our convention is that Bok Choy tests have a BDD-style docstring explaining what it is that is being tested.

You can see other examples in this file, e.g. https://github.com/edx/edx-platform/blob/master/common/test/acceptance/tests/lms/test_lms_courseware.py#L215